tests/e2e/specs/out-of-game/lobby.spec.js

```diff
@@ -149,9 +149,10 @@ describe('Lobby - P0 Perspective', () => {
       assertGameStarted();
     });
   });
-  it.skip('[Missing Feature] Loads lobby after page refresh', () => {
+  // TODO: Rewrite this test when auth is supported in the lobby
+  it.skip('Redirects to game list after page refresh', () => {
     cy.reload();
-    expect(true).to.eq(false, 'Empty Test');
+    cy.hash().should('equal', '#/');
```

Comment:
If we want to be a bit more thorough here we could assert that you can then join a game, see yourself as the only player, have an opponent join, and see that the game starts normally

Reply:
That makes sense. Not sure the right way to balance the overhead of redundant test flows against the potential confidence that that same logic works on refresh. I like your idea of checking a full happy flow after reconnect though then there is the consideration of which reconnect situations to fully check.

Either way I think this is fine for now